src/test/unit/algorithms/SpatialPoolerTest.cpp

```diff
@@ -172,65 +172,83 @@ namespace nupic {
               sp2.getMinPctActiveDutyCycles()));
 
 
-    Real boostFactors1[numColumns];
-    Real boostFactors2[numColumns];
+    auto boostFactors1 = new Real[numColumns];
```

Comment:
Oh right, I didn't realize it was initializing the array with a dynamic value. This is fine.